contribution-guide.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: ballerina-inner-page
+title: The Ballerina Contribution Guide
+permalink: /contribution-guide/
+---
+
+# The Ballerina Contribution Guide
+
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is licensed under the Apache license and is nurtured by all the contributors of the Ballerina community.
+
+We appreciate your help!
+
+- [Get started](#get-started)
+- [Set up the Prerequisites](#set-up-the-prerequisites)
+- [Set up the developer environment](#set-up-the-developer-environment)
+    - [Set up IntelliJ IDEA](#set-up-intellij-idea)
+    - [Set up Eclipse](#set-up-eclipse)
+- [Obtain the source code](#obtain-the-source-code)
+- [Build the project](#build-the-project)
+- [Contribute to Ballerina grammar](#contribute-to-ballerina-grammar)
+- [Submit your contribution](#submit-your-contribution)
+- [Accept the Contributor License Agreement](#accept-the-contributor-license-agreement)
+- [Propose changes to Ballerina](#propose-changes-to-ballerina)
+
+## Get started
+
+- Join the conversations at:
+
+    - [Ballerina-Dev Google Group](https://groups.google.com/forum/#!forum/ballerina-dev): to join the developer team mailing list to discuss Ballerina roadmap, features, issues that are in progress etc.
+    - [StackOverflow](https://stackoverflow.com/questions/tagged/ballerina): to get help with Ballerina. (Use the Ballerina tag for your questions there.) 
+    - [Slack](https://app.slack.com/client/T47EAEKB5/DF818FX9Q): for real-time communications with the team and community.
```

Comment:
Is this link correct for slack? Better to put [1] without the client redirected link.

[1] https://ballerina-platform.slack.com/

Reply:
Changed the link accordingly.